lib/Core/StatsTracker.cpp

```diff
@@ -454,6 +454,7 @@ void StatsTracker::writeStatsHeader() {
          << "WallTime REAL,"
          << "CoveredInstructions INTEGER,"
          << "UncoveredInstructions INTEGER,"
+         << "PostLCPTime INTEGER,"
```

Comment:
Fine to be an integer here; the milliseconds to seconds (integer to float) conversion occurs later in `klee-stats` itself.